Content.Client/GameObjects/Components/Construction/ConstructorComponent.cs

```diff
@@ -33,9 +34,9 @@ public override void Initialize()
             Owner.GetComponent<ITransformComponent>();
         }
 
-        public override void HandleMessage(ComponentMessage message, INetChannel netChannel = null, IComponent component = null)
+        public override void HandleNetworkMessage(ComponentMessage message, INetChannel channel, ICommonSession session = null)
```

Comment:
`PlayerAttachedMsg` and `PlayerDetachedMsg` are raised locally so you'll need to separate these out.